eras/mary/impl/cardano-ledger-mary.cabal

```
@@ -102,7 +102,7 @@ library testlib
         cardano-crypto-class,
         cardano-ledger-binary:testlib,
         cardano-ledger-core:{cardano-ledger-core, testlib},
-        cardano-ledger-mary >=1.0.0.1,
+        cardano-ledger-mary >=1.4.0.1,
```

This bound need to be completely removed. It is for this package, which makes no sense at all and should have never been here to begin with.
```suggestion
        cardano-ledger-mary,
```